staging/src/k8s.io/kms/internal/plugins/base64/kms.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: base64-kmsv2-provider
+  namespace: kube-system
+  labels:
+    tier: control-plane
+    component: base64-kmsv2-provider
+spec:
+  hostNetwork: true
+  containers:
+    - name: base64-kmsv2-provider
+      image: localhost:5000/base64:e2e
+      imagePullPolicy: IfNotPresent
+      volumeMounts:
+        - name: sock
+          mountPath: /tmp
```

Comment:
Let's not use `/tmp` for the socket. Use `/var/run`.

Reply:
`/var/run` results in permission denied since we are using nonroot image. Also considering this is the mock service I thought `/tmp` is the apt location.

Reply:
Temp directories and their contents are ephemeral which doesn't sound appropriate for a socket/file we care about.